native/src/main/java/io/ballerina/stdlib/mime/util/EntityBodyHandler.java

```diff
@@ -456,6 +509,17 @@ public static BStream getByteStream(BObject entityObj) {
                 (ENTITY_BYTE_STREAM) : null;
     }
 
+    /**
+     * Obtains the byte stream if the content type is text/event-stream.
+     *
+     * @param entityObj Represent a ballerina entity
+     * @return A Ballerina byte stream
+     */
+    public static BStream getEventStream(BObject entityObj) {
```

Comment:
Do we need to expose them in the ballerina side? (like https://github.com/ballerina-platform/module-ballerina-mime/blob/abf505a3c70d3a095f70f801cf5db8023903d8b6/ballerina/natives.bal#L451)

Reply:
Resolving this comment as per our offline code review discussion where we decided not to include a ballerina API.